ProcessApplicationFormFunction/Database/Models/AcademyConversionProject.cs

```diff
@@ -4,44 +4,46 @@
 
 namespace ProcessApplicationFormFunction.Database.Models
 {
-    [Table("AcademyConversionProject", Schema = "sdd")]
-    public record AcademyConversionProject
-    {
-        [Key]
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
-        public int Id { get; set; } 
-        public int IfdPipelineId { get; set; } 
-        public int? Urn { get; set; }
-        public string SchoolName { get; set; }
-        public string LocalAuthority { get; set; }
-        public string ApplicationReferenceNumber { get; set; }
-        public string ProjectStatus { get; set; }
-        public DateTime? ApplicationReceivedDate { get; set; }
-        public DateTime? OpeningDate { get; set; }
-        public string TrustReferenceNumber { get; set; }
-        public string NameOfTrust { get; set; }
-        public string SponsorReferenceNumber { get; set; }
-        public string SponsorName { get; set; }
-        public string AcademyTypeAndRoute { get; set; }
-        public DateTime? ProposedAcademyOpeningDate { get; set; }
+   [Table("Project", Schema = "academisation")]
```

Comment:
Is there a situation where we might want a feature flag here or are we happy that it's all fed to the new schema going forwards?

Reply:
Good question. Tbh I think it might be overkill for this because this function is not regularly changing and we are unlikely to need to release additional changes before this goes out.
There is an argument to say we should continue creating a record in the sdd schema as well. So effectively we could create two projects one in academisation & another in sdd. What do you think?